CHANGELOG.md

```diff
@@ -5,9 +5,9 @@ All notable changes to this project will be documented in this file.
 
 ## 2.7.0
 
-- Update grafana-aws-sdk version
+- Update grafana-aws-sdk version to include new region in opt-in region list https://github.com/grafana/grafana-aws-sdk/pull/80 
 - Security: Upgrade Go in build process to 1.20.4
-- Revert grafana-plugin-sdk-go version to 0.149.0 to fix connection issues. Should be same behavior as behavior with no known issues.
+- Update grafana-plugin-sdk-go version to 0.161.0 to fix connection issues.
```

Comment:
Just a doubt here since we aren't yet aware of any connection issues in any plugin other than Athena.
Another idea would be to say that we are using 0.161.0 in order to not forward HTTP headers by default https://github.com/grafana/grafana-plugin-sdk-go/pull/679